emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java

```diff
@@ -136,7 +136,11 @@ public boolean knows(EClassifier eClassifier) {
     return null;
   }
 
-  public void populateInternal() {
+  public void prePopulateBuiltins() {
+    ePackagesToLanguages.put(
+        BuiltinsEMFPackageProvider.getEPackage(), LionCoreBuiltins.getInstance());
```

Comment:
What does `BuiltinsEMFPackageProvider` do?

Reply:
I should have removed that. This was the thing I was using before getting the Builtins EMF Package from your PR. Removed